zerver/views/users.py

```diff
@@ -527,14 +527,9 @@ def get_members_backend(
     user_profile: UserProfile,
     user_id: Optional[int] = None,
     include_custom_profile_fields: bool = REQ(json_validator=check_bool, default=False),
-    client_gravatar: bool = REQ(json_validator=check_bool, default=False),
+    client_gravatar: bool = REQ(json_validator=check_bool, default=True),
 ) -> HttpResponse:
-    """
-    The client_gravatar field here is set to True if clients can compute
-    their own gravatars, which saves us bandwidth.  We want to eventually
```

Comment:
Maybe this comment should be move to somewhere else explaining why `True` is the default?

Reply:
The first sentence is correct so maybe we can keep that. I'll restore that.